src/tests/ftest/util/ior_utils.py

```diff
@@ -264,6 +264,22 @@ def get_ior_metrics(cmdresult):
 
         return (write_metrics, read_metrics)
 
+    @staticmethod
+    def log_metrics(logger, message, metrics):
+        """Log the ior metrics
+
+           Args:
+               logger (log): logger object handle
+               message (str) : Message to print before logging metrics
+               metric (lst) : IOR write and read metrics
+        """
+        logger.info("\n")
+        logger.info(message)
+        logger.info(metrics[0])
```

Comment:
Would it make sense to make this a bit more flexible by doing:
`for m in metrics:
    logger.info(metrics[m])
`